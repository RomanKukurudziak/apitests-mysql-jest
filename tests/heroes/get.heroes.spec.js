const { restoredb, api } = require('../helpers');

describe('GET /heroes should work correct', () => {

    beforeAll(async () => await restoredb())

    it('Returns  correct with correct params', async () => {
        const { data } = await api.get(`/heroes`);
        expect(data).toMatchSnapshot();
    }); 
});