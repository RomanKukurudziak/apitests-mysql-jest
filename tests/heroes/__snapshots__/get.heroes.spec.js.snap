// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GET /heroes should work correct Returns  correct with correct params 1`] = `
{
  "list": [
    {
      "createdAt": "2022-10-07T13:42:31.000Z",
      "id": 1,
      "name": "Arc Warden",
      "ownerId": 13,
      "updatedAt": "2022-10-07T13:42:31.000Z",
    },
    {
      "createdAt": "2022-10-07T13:42:58.000Z",
      "id": 2,
      "name": "Drow Ranger",
      "ownerId": 4,
      "updatedAt": "2022-10-07T13:42:58.000Z",
    },
    {
      "createdAt": "2022-10-07T13:43:22.000Z",
      "id": 3,
      "name": "Phantom Assassin",
      "ownerId": 7,
      "updatedAt": "2022-10-07T13:43:22.000Z",
    },
  ],
}
`;
